package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@okta/okta-signin-widget",
   "description": "The Okta Sign-In Widget",
-  "version": "5.5.0",
+  "version": "5.6.0",
```

Comment:
I don't think you need to upgrade this version here..

Reply:
Ah good catch! This was the result of a sloppy rebase

Reply:
I removed this commit